.travis.yml

```diff
@@ -9,9 +9,14 @@ d:
  - ldc-beta
  - ldc
 
+addons:
+  apt:
+    packages:
+      - libevent-dev
```

Comment:
I thought that `libevent-dev` is not needed anymore now that we're using `vibe-d:core` in configuration `vibe-core`?

Reply:
I thought so too, but I was hitting linking errors on Travis. I then concluded that getting the build to pass is the first priority.

Reply:
I see, thanks for explaining.